app-admin/passwordsafe/passwordsafe-1.16.0.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 2023 Gentoo Authors
```

Comment:
should we here keep the Copyright Year (2021-2023)? since this ebuild is mostly a copy-and-paste

Reply:
Just looked at https://devmanual.gentoo.org/ebuild-writing/file-format/#ebuild-header where it states:

... This must be an exact copy of the contents of[ header.txt](https://gitweb.gentoo.org/repo/gentoo.git/tree/header.txt) in the top directory of the Gentoo repository.

So I my opinion is a single copyright year is that what is intended by the manual, although header.txt still contains 2022.

Reply:
the header.txt is already updated to 2023, but I think it's the new package (not exist in portage before) added that should use the exact copy of header.txt, while for the ebuild inherited from old versions should preserve copyright years..

I haven't found explicit policy on this, and do not have strong position here, so will leave other developer for review

Reply:
Yeah I was reading our copyright policy GLEP. IIRC we **allowed** a single-year copyright for **new** files, but any old files **need** to retain the old copyright information. In this case, if this ebuild was written from **scrath** you can keep 2023, otherwise add the previous years too.

Either case, you shouldn't manually edit the line, you should let **pkgdev** handle it for you.